src/components/include/cos_asm_server_stub_simple_stack.h

```diff
@@ -42,6 +42,33 @@ name##_inv:                       \
                                   \
         sysenter;
 
+#define cos_asm_server_stub_3rets(name) \
+.globl name##_inv ;                     \
+.type  name##_inv, @function ;	        \
+.align 16 ;			        \
+name##_inv:                             \
+        COS_ASM_GET_STACK               \
```

Comment:
I'm looking at this for the first time on github. Spacing looks so silly.
I'll fix it and also in the other addition to this file.